repos/system_upgrade/el7toel8/actors/storagescanner/libraries/library.py

```diff
@@ -79,22 +80,46 @@ def _get_fstab_info(fstab_path):
     ''' Collect storage info from /etc/fstab file '''
     if _is_file_readable(fstab_path):
         with open(fstab_path, 'r') as fstab:
-            for entry in fstab:
+            keys = ['fs_spec', 'fs_file', 'fs_vfstype', 'fs_mntops', 'fs_freq', 'fs_passno']
+            for line, entry in enumerate(fstab, 1):
                 if entry.startswith('#'):
                     continue
 
+                if '#' in entry:
+                    api.current_logger().warning(
+                        'Detected in /etc/fstab a "#" comment in line %s please verify if it is correct.',
+                        line
+                    )
+
                 entry = entry.strip()
                 if not entry:
                     continue
```

Comment:
That's something that can be done safelly in the beginning of the loop. Whitespaces are not something that would bother us even in case of comments.